zaza/openstack/charm_tests/tempest/templates/tempest_v3.j2

```diff
@@ -164,3 +168,34 @@ nameservers = {{ test_name_server }}
 [service-clients]
 # Default is 60s which is not enough for resource constrained environments.
 http_timeout = 120
+
+{% if enforce_scope is defined and enforce_scope -%}
+[enforce_scope]
+{% if "keystone" in enabled_services -%}
```

Comment:
This would be more compact if we use the same approach used by the `[service_available]` section

```
{% for svc in enabled_services -%}
{{ svc }} = true
{% endfor -%}
{% for svc in disabled_services -%}
{{ svc }} = false
{% endfor %}
```

Reply:
Indeed but heat tempest plugin, for example, does support this. 

Should I create a new context setting with all the compatible plugins and service enabled?

Reply:
yeah, I think that would be better, that way is easier to add others as they grow support